src/ReactiveDomain.Policy.Tests/PolicyLoadTests.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace ReactiveDomain.Policy.Tests
+{
+    public class PolicyLoadTests
+    {
+        [Fact]
+        public void can_parse_args() { 
+        
+        }
```

Comment:
```
        public void can_parse_args() { 
            throw new NotImplementedException();
        }
```
OR
```
        public void can_parse_args() { 
            // TODO: implement
        }
```

Reply:
from an earlier implementation, parsing is handled by `System.CommandLine` now